modules/core/src/lib/api/select-loader.ts

```diff
@@ -76,7 +76,7 @@ export function selectLoaderSync(
   }
 
   // Add registered loaders
-  loaders = [...(loaders || []), ...getRegisteredLoaders()];
+  loaders = [...((loaders as Loader[]) || []), ...getRegisteredLoaders()];
```

Comment:
We could create a new variable that is guaranteed to be an array.

Reply:
I tried that but that still raises `@typescript-eslint/no-unnecessary-type-assertion`, so I figured I might as well do it like this.